### PR TITLE
Document loopback redirect URL in identity API

### PR DESCRIPTION
**Pull request by Rob--W**
https://bugzilla.mozilla.org/show_bug.cgi?id=1635344

### CODE REVIEW

**Comment by Rob--W** [comment]
I wanted to add @mixedpuppy as a reviewer but am somehow not able to.

**Comment by chrisdavidmills** [comment]
> I wanted to add @mixedpuppy as a reviewer but am somehow not able to.

I think this is because Shane hasn't accepted the invitation to this repo's collaborators list. It is still listed as "pending". I can resend the invite if that would help.

**Comment by mixedpuppy** [comment]
> > I wanted to add @mixedpuppy as a reviewer but am somehow not able to.
> 
> I think this is because Shane hasn't accepted the invitation to this repo's collaborators list. It is still listed as "pending". I can resend the invite if that would help.

I found the invite, but it seems like it no longer works.

**Comment by chrisdavidmills** [comment]
> > > I wanted to add @mixedpuppy as a reviewer but am somehow not able to.
> > 
> > 
> > I think this is because Shane hasn't accepted the invitation to this repo's collaborators list. It is still listed as "pending". I can resend the invite if that would help.
> 
> I found the invite, but it seems like it no longer works.

No worries; I've just tried resending it; let me know if you still have trouble.

**Comment by Ryuno-Ki** [comment]
> I found the invite, but it seems like it no longer works.

I had that, too. Normally, the invitation expires after seven days automatically.

File: files/en-us/mozilla/add-ons/webextensions/api/identity/index.html [hunk 1]
```diff
@@ -32,12 +32,17 @@ <h2 id="Setup">Setup</h2>
...
 
 <p>You get the redirect URL by calling {{WebExtAPIRef("identity.getRedirectURL()")}}. This function derives a redirect URL from the add-on's ID.  To simplify testing, set your add-on's ID explicitly using the <code><a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings">browser_specific_settings</a></code> key (otherwise, each time you <a href="/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox">temporarily install the add-on</a>, you get a different redirect URL).</p>
 
+<p>{{WebExtAPIRef("identity.getRedirectURL()")}} returns a URL at a fixed domain name and a subdomain derived from the add-on's ID. Some OAuth servers (such as Google) only accept domains with a verified ownership as the redirect URL. As the dummy domain cannot be controlled by extension developers. the default domain cannot always be used.
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/api/identity/index.html:39]: `developers, `

**Review by mixedpuppy — CHANGES_REQUESTED** [review]
looks fine, just a comma

File: files/en-us/mozilla/add-ons/webextensions/api/identity/index.html (hunk 1 above)
Comment (Rob--W) [reply files/en-us/mozilla/add-ons/webextensions/api/identity/index.html]: Thanks, done.

**Review by mixedpuppy — APPROVED** [review]

**Pull request merged**